admin/album.php

```diff
@@ -217,6 +217,18 @@
     ),
   'public_physical' => array(
       'name' => l10n('From public physical albums'),
+      'sif_lnk' => ($conf['SmartAlbums']['smart_is_forbidden'])?'Smart is forbidden':'Smart is allowed',
+      'sif' => ($conf['SmartAlbums']['smart_is_forbidden'])?'visitor permissions prepend. Pictures from private albums will be rejected for ungranted visitors. Hence, <strong>using this filter is not required</strong>.':'static permissions. Pictures from private albums may be associated to this virtual smart album. <strong>Using this filter will prevent this to happen</strong>.',
```

Comment:
In order to provide information to SmartAlbums building page. Wish it helps, but feel it somehow heavy. Need improvements.